doc/introduction/interfaces/numpy.rst

```diff
@@ -33,13 +33,13 @@ Via the QNode decorator
 
 The :ref:`QNode decorator <intro_vcirc_decorator>` is the recommended way for creating QNodes
 in PennyLane. By default, all QNodes are constructed for the NumPy interface,
-but this can also be specified explicitly by passing the ``interface='numpy'`` keyword argument:
+but this can also be specified explicitly by passing the ``interface='autograd'`` keyword argument:
 
 .. code-block:: python
 
     dev = qml.device('default.qubit', wires=2)
 
-    @qml.qnode(dev, interface='numpy')
+    @qml.qnode(dev, interface='autograd')
```

Comment:
(docstrings have long favoured using `'autograd'` over `'numpy'`, so I've made the update here as well)